akka-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: akka-operator
-description: Akka Operator
+description: Akka Operator in Akka Cloud Platform
```

Comment:
Akka Cloud Platform Operator?

Reply:
could work but it's a mouthful. In docs I have used Akka Operator when referring specifically to the operator and Akka Cloud Platform for the "bigger" product picture. I'm thinking that the Akka Cloud Platform can contain more than the Akka Operator in the future.

Reply:
Would be nice to have one operator, (not have many) and of course the product can grow over time, we can always rename.

Reply:
I would have liked "Akka Cloud" as a product name but I can understand the Platform addition, to not make it sound like we are doing a full cloud service like Akka serverless...

Reply:
we are speculating, it could be other things than the operator, for example something for Telemetry

Reply:
Oh I just meant that the name `Akka Cloud Platform Operator` would be problematic if there was more than one operator for Akka Cloud Platform, I thought you meant that.